openwrt/Makefile

```diff
@@ -12,6 +12,8 @@ include include/*.mk
 TARGET ?= ar71xx
 
 BUILD_DIR ?= build
+# Set to configure -j (good rules is equal to number of cores on machine)
+JOBS ?= 2
```

Comment:
We can do better about inferring `-j`, but would like to get these tested in a followup